.github/workflows/check-bindings.yaml

```diff
@@ -0,0 +1,78 @@
+---
+name: Check Contract Bindings
+
+on:
+  merge_group:
+    types: [checks_requested]
+  pull_request:
+    types:
+      - synchronize
+      - ready_for_review
+
+jobs:
+  check-bindings:
+    runs-on: self-hosted-hoprnet-bigger
```

Comment:
I don't think this requires the bigger runner, right?

Reply:
changed to big. Tried with small but it took too long to complete